TaylorFramework/Modules/temper/Rules/CyclomaticComplexityRule.swift

```diff
@@ -25,7 +25,7 @@ final class CyclomaticComplexityRule : Rule {
         }
     }
     private let decisionalComponentTypes = [ComponentType.If, .While, .For, .Case, .ElseIf, .Ternary, .NilCoalescing, .Guard]
-    
```

Comment:
Why did we touch all these white spaces?  
If someone is intending from Xcode(control+I), they will appear again.